Week3/moreJs-9.js

```diff
@@ -0,0 +1,20 @@
+/*
+Change the function vehicle to use the list of question 7.
+ So that vehicle("green", 3, 1) prints "a green new bike".
+*/
+
+function vehicle (color, type, age){
+    let vehicles = ['motorbike' , 'caravan', 'bike', 'car'];
+    // variable 'vehicles' is can be read from  moreJs-7and8.js
+    // but i used it again inside of function to show you,
+    // and in case you can run in a html seperately.
+    console.log('a ' + color +' ' + condition(age) + vehicles[(type-1)]);
+}
+function condition(years){
```

Comment:
Maybe the name age here is more consistent, because you use it aswell above

Reply:
yes. you re right. 
I was trying to find some other parameters name. I thought it might confuse ('age' in 2 places).
but I forgot about scopes, and parameters I think.
I changed it to age. its better